cdk/autopilot/mapping/identifyMapping.yaml

```diff
@@ -0,0 +1,39 @@
+- destKey: Email
+  sourceKeys:
+  - traits.email
+  - context.traits.email
+  required: false
```

Comment:
Keep `required: false` as the default behavior and set it here to override that.

Reply:
That is the default behavior here we were just explicit. But yeah, we can remove it here